speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backendUnitTests/MediatorTests.cpp

```diff
@@ -531,18 +531,18 @@ TEST(MediatorTests, SetAccelDecisionTest)
     {
         bool accelBool = random.NextBool();
         SDAConfigMediator::GetInstance()->SetAccelDecision(accelBool);
-        ASSERT_EQ(SDAConfigMediator::GetInstance()->IsAccelDecision(), accelBool);
+        ASSERT_EQ(SDAConfigMediator::GetInstance()->HasMadeAccelDecision(), accelBool);
     }
 }
 
-/// @brief tests if the accel decision is correctly set
+/// @brief tests if the user can use the controls at the correct moment
 TEST(MediatorTests, CanUseTest)
 {
     SDAConfigMediator::ClearInstance();
     ASSERT_TRUE(SetupSingletonsFolder());
     Random random;
 
-    for (int i = 0; i < TEST_AMOUNT; i++)
+    for (int i = 0; i < 2 * TEST_AMOUNT; i++)
```

Comment:
Rather not just do these tests twice as much to get all combinations covered. Pairwise tests or something would be better